### PR TITLE
Update main readme

### PR DESCRIPTION
**Pull request by fgao22**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+#### Library we are using: 
+[Mozzi](https://sensorium.github.io/Mozzi/) to generate algorithmic music on an arduino board
+
+[Three.js](https://threejs.org/) for 3D object visualization
+
+[Websocket.io](http://websocket.io/) for sending music data from arduino to raspberry pi
+
+[Node.js](https://nodejs.org/en/) for backend webserver
+
```
Comment (fgao22) [inline README.md:24]: Add here if we used more libraries

File: README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+## Expected Parts:
+* Speaker
+* Buttons
+* Wires
+* Arduino
+* Raspberry Pi
+* Potentiometer
+* Accelerometer
+* Flex Sensor
+* Pressure Sensor
```
Comment (fgao22) [inline README.md:34]: This list is prob not comprehensive

File: README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Interaction Plan:
+Upon connection, the arduino will output sound signal. The user can interact with all the included sensors (detials see following) to modify and distort the sound and hear it from the speaker. The user can also view a music visualization from the browser that interacts with user input in real-time.
+
+#### Functions of sensors:
```
Comment (fgao22) [inline README.md:39]: Yuxin or Christine is way better explaining this than me :)

**Review by cku3 — APPROVED** [review]
Looks good as a start, we can add in more details in a bit!

**Pull request merged**